.github/workflows/review_pull_request.yml

```diff
@@ -10,10 +10,10 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     steps:
-      - name: Set up JDK 1.8
+      - name: Set up JDK 11
         uses: actions/setup-java@v1
         with:
-          java-version: 1.8
+          java-version: 11
```

Comment:
There are a few more files that require this change too 👍 

- `.cicdtemplate/.github/workflows/deploy_staging_and_production_to_firebase_app_distribution.yml`
- `.cicdtemplate/.github/workflows/review_pull_request.yml`
- `.cicdtemplate/.github/workflows/run_detekt_and_unit_tests.yml`

Reply:
@Tuubz Good catch 👍 I will update 🚀